source/install/setting-up-local-machine-using-docker.rst

```diff
@@ -34,101 +34,97 @@ If you have Docker set up, Mattermost installs in one line:
+         docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview
 
-1. Follow the instructions at https://docs.docker.com/engine/installation/linux/fedora/ or use the summary below:
+    3. When Docker is done fetching the image, open ``http://localhost:8065/`` in your browser.
+  
+  .. tab:: Fedora
+  
+    1. Follow the `Install Docker Engine on Fedora <https://docs.docker.com/engine/installation/linux/fedora/>`__ documentation, or follow the summary steps below:
 
-.. code:: bash
+      .. code:: bash
       
-   sudo dnf -y install dnf-plugins-core
-   sudo dnf config-manager \
-   --add-repo \
-   https://download.docker.com/linux/fedora/docker-ce.repo
-   sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
-   sudo usermod -aG docker <username>
-   sudo systemctl start docker
+         sudo dnf -y install dnf-plugins-core
+         sudo dnf config-manager \
+         --add-repo \
+         https://download.docker.com/linux/fedora/docker-ce.repo
+         sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
+         sudo usermod -aG docker <username>
+         sudo systemctl start docker
  
-2. Start Docker container: 
+    2. Start Docker container: 
 
-.. code:: bash
+      .. code:: bash
       
-   docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview
+         docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview
        
-3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
-
-Arch Linux
-~~~~~~~~~~
-
-To install the preview on Arch Linux, see the `installation guide <https://wiki.archlinux.org/index.php/Mattermost#With_Docker>`__ on the Arch Linux wiki.
+    3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
+  
+  .. tab:: Arch Linux
+  
+    To install the preview on Arch Linux, see the `installation steps <https://wiki.archlinux.org/index.php/Mattermost#With_Docker>`__ on the Arch Linux wiki.
 
 Setting up SMTP email (Recommended) 
 -----------------------------------
 
-The default single-container Docker instance for Mattermost is designed for product evaluation, and sets ``SendEmailNotifications=false`` so the product can function without enabling email. To see the product's full functionality, enabling SMTP email is recommended.
-
-.. include:: ../configure/smtp-email.rst 
-  :start-after: How to Enable Email
+The default single-container Docker instance for Mattermost is designed for product evaluation, and sets ``SendEmailNotifications=false`` so the product can function without enabling email. To see the product's full functionality, we recommend enabling SMTP email. See the `SMTP Email Setup <https://docs.mattermost.com/configure/smtp-email.html>`__ documentation for details.
```

Comment:
**product's email notification functionality** is more suitable. Email notifications are just a small feature of the product and not necessary to spin up the preview.

Reply:
@mrckndt - I'm a bit unclear about your feedback here. Can you please elaborate on what you'd like to see changed in this section of the product documentation? Thanks!

Reply:
@cwarnermm 
> To see the product's full functionality

I think it's a bit misleading because the email notifications are only relevant for production deployments and the absence isn't really blighting the chat and Mattermost experience in general.